smart_campus/app/views.py

```diff
@@ -1,3 +1,140 @@
 from django.shortcuts import render
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+from django.contrib.auth import authenticate, login, logout
+from django.http import HttpResponseRedirect, JsonResponse
+from django.contrib.auth.decorators import login_required
 
-# Create your views here.
+from .models import (
+    User, Reward, Permission,
+    Station,
+)
+
+
+@csrf_exempt
+@require_POST
+def signup(request):
+    """
+    API
```

Comment:
Following [Google Style Python Docstring](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), the first line should be a brief description of this function.
`API` is not a suitable description.